src/seedu/addressbook/common/UtilsTest.java

```diff
@@ -0,0 +1,37 @@
+package seedu.addressbook.common;
+
+import static org.junit.Assert.*;
+import org.junit.Test;
+import java.util.*;
+
+public class UtilsTest {
+
+	@Test
+	public void testIsNull() {
+		assertTrue(Utils.isAnyNull( (Object) null ));
+	}
+
+	@Test
+	public void testIsNotNull() {
+		assertFalse(Utils.isAnyNull( new Object() ));
+	}
+
+	@Test
+	public void testIsNotUnique() {
+		ArrayList<Integer> test = new ArrayList<Integer>();
+		for (int i = 0; i < 5; i++){
+			test.add(1);
+		}
```

Comment:
You can use `List<Integer> list = Arrays.asList(1, 1, 1, 2)` to create and initialize a list in one line.